students/cindywaldron/lesson4/mailroom.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+from datetime import date
+
+donors = {'William Gates, III': [653784.49, 2, 326892.24],
+    'Mark ZuckerBerg': [16396.10, 3, 5465.37],
+    'Jeff Bezos': [877.33, 1, 877.33],
+    'Paul Allen': [708.42, 3, 236.14] }
+
+options = { 1: 'Send Thank You', 2: 'Create a Report', 3:'Send letters to everyone', 4:'Quit'}
+
+def getKey(items):
```

Comment:
Per PEP8 function names should be lower case with underscores separating works, e.g., get_key